docs/source/architecture/journal.rst

```diff
@@ -310,8 +311,7 @@ If any transactions in the genesis block set or change settings, Sawtooth
 requires the `Sawtooth Settings transaction processor <../cli/settings-tp>`_
 or an equivalent implementation.
 For example, if the genesis block configures PoET consensus, this transaction
-processor is handles the transactions with PoET settings. (If no
-consensus is specified, Sawtooth uses dev mode consensus.)
+processor is handles the transactions with PoET settings.
```

Comment:
Can you also remove the "is" from the previous sentence?

Reply:
I saw that. The mistake is still in the master branch, so I left it there.

So I fixed it. The error remains in the master branch.